content/client-lib-development-guide/features.textile

```diff
@@ -2141,6 +2152,10 @@ class VCDiffDecoder
 class DeltaExtras
   from: String // the id of the message the delta was generated from
   format: String //the delta format. Only vcdiff is supported as at API version 1.2
+
+class InteractionExtras:
```

Comment:
I mentioned before - we should also update `TM2i` with something similar to the sentence about `DeltaExtras`

Reply:
TM2i definitely needs to be reworked into something more readable but for now I've added it.

Reply:
Yeah agreed, but obvs beyond the scope of this work